src/main/java/org/dependencytrack/persistence/jdbi/FindingDao.java

```diff
@@ -386,7 +387,7 @@ default PaginatedResult getAllFindings(final AlpineRequest alpineRequest, final
     @AllowUnusedBindings
     @DefineNamedBindings
     @RegisterConstructorMapper(GroupedFindingRow.class)
-    List<GroupedFindingRow> getGroupedFindings(@Define String queryFilter, @Define boolean activeFilter);
+    List<GroupedFindingRow> getGroupedFindings(@Define String queryFilter, @Define String aggregateFilter, @Define boolean activeFilter);
```

Comment:
Please keep the separation between query and parameters. Directly embedding user-provided values into the query has a high risk of SQL injection. Did you try using [`@BindMap`](https://jdbi.org/#_bind_map_instances_with_bindmap) to pass the `params` map?